packages/@aws-cdk/aws-appmesh/lib/validation-context.ts

```diff
@@ -0,0 +1,91 @@
+import * as cdk from '@aws-cdk/core';
+import { CfnVirtualNode } from './appmesh.generated';
+import { ACMTrustOptions, FileTrustOptions } from './client-policy';
```

Comment:
What's the point of the split into 2 files, `validation-context.ts` and `client-policy.ts`? It looks to me like they should be all in one file...